configurations/operator/perf-regression-latency-and-throughput-multitenant-14-clients.yaml

```diff
@@ -0,0 +1,144 @@
+
+prepare_write_cmd: [[
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..5000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=5000001..10000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=10000001..15000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..20000000",
+], [
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..5000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=5000001..10000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=10000001..15000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..20000000",
+], [
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..5000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=5000001..10000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=10000001..15000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..20000000",
+], [
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..5000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=5000001..10000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=10000001..15000000",
+    "cassandra-stress write no-warmup cl=ALL n=5000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=2 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=15000001..20000000",
```

Comment:
IIUC, each cluster writes a dataset of a ~20GB, right?
It probably fits the entire cache, is it?

Reply:
14 clients config was not really used for perf testing and comparison. Only 2 and 7 clients were.
So, I have no answer for your question here.

Reply:
The stress commands repeats themselves in all cases, aren't they?

Reply:
We have here 2 different combinations. It is config for both perf testing kinds - latency and thrpt.